generators/server/templates/src/main/java/package/config/MetricsConfiguration.java.ejs

```diff
@@ -111,6 +111,8 @@ public class MetricsConfiguration extends MetricsConfigurerAdapter {
         <%_ if (databaseType === 'sql') { _%>
         if (hikariDataSource != null) {
             log.debug("Monitoring the datasource");
+            //remove the factory create by HikariDataSourceMetricsPostProcessor until jhipster migrate to micrometer
```

Comment:
Please polish this comment :
- space before remove
- create ->created
- jhipster -> JHipster
- micrometer -> Micrometer